### PR TITLE
toml-test 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by awvwgk**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/toml-test.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  url "https://github.com/BurntSushi/toml-test/archive/refs/tags/v1.2.0.tar.gz"
+  sha256 "33bf4e9c017cd57f3602e72d17f75fb5a7bcc7942541c84f1d98b74c12499846"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+
+    (share/"toml-test").install "tests"
```
Comment (carlocab) [inline Formula/toml-test.rb:13]:
```suggestion
    system "go", "build", *std_go_args
    (share/"toml-test").install "tests"
```

File: Formula/toml-test.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  end
+
+  test do
+    system "#{bin}/toml-test", "-version"
+    system "#{bin}/toml-test", "-help"
```
Comment (carlocab) [inline Formula/toml-test.rb:18]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Comment (awvwgk) [reply Formula/toml-test.rb:17]: Done.

**Review by carlocab — APPROVED** [review]
Thanks!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@awvwgk, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**